Library/Formula/ddar.rb

```diff
@@ -8,11 +8,14 @@ class Ddar < Formula
   head "https://github.com/basak/ddar.git"
 
   depends_on "xmltoman" => :build
-  depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on :python
```

Comment:
Tim will be able to talk more about the upsides/downsides (if he has time & desire to), but Python really isn't a small dependency and Homebrew was built on the principle of using the system tools available. That's developed over time to insert caveats about using system tools available *providing* they are secure and not chronically neglected (see deprecating Apple's OpenSSL), but I'm not sure Python falls under that bracket.

Tim's the Python guru though; I trust his thoughts on these various discussions.